### PR TITLE
Sprint1308 3008/change station list view

### PR DESCRIPTION
**Pull request by aydin101**

### CODE REVIEW

File: ui_src/src/services/valueConvertor.js [hunk 1]
```diff
@@ -99,6 +99,27 @@ export const parsingDate = (date, withSeconds = false, withTime = true) => {
         return '';
     }
 };
+
+export const parsingDateTime = (dateString) => {
```
Comment (avrhamNeeman) [inline ui_src/src/services/valueConvertor.js:103]: We already have "parsingDate" function; please use it.

File: ui_src/src/assets/images/timeSchedule.svg
Comment (avrhamNeeman) [inline ui_src/src/assets/images/timeSchedule.svg:1]: Please delete the current retention icon. (retentionIcon.svg)

File: ui_src/src/assets/images/TimeFill.svg
Comment (avrhamNeeman) [inline ui_src/src/assets/images/TimeFill.svg:1]: Change it to lowercase, please

File: ui_src/src/domain/stationsList/stationBoxOverview/index.js [hunk 2]
```diff
@@ -28,8 +28,12 @@ import replicasIcon from '../../../assets/images/replicasIcon.svg';
 import totalMsgIcon from '../../../assets/images/totalMsgIcon.svg';
 import poisonMsgIcon from '../../../assets/images/poisonMsgIcon.svg';
 import remoteStorage from '../../../assets/images/remoteStorage.svg';
+import { ReactComponent as ClockIcon } from '../../../assets/images/TimeFill.svg';
```
Comment (avrhamNeeman) [inline ui_src/src/domain/stationsList/stationBoxOverview/index.js:31]: You can import it in a more readable way:
import ClockIcon  from '../../../assets/images/TimeFill.svg';

File: ui_src/src/domain/stationsList/stationBoxOverview/index.js [hunk 3]
```diff
@@ -56,38 +60,29 @@ const StationBoxOverview = ({ station, handleCheckedClick, isCheck }) => {
...
+
+        let outputString = inputString;
+        for (const replacement of replacements) {
+            outputString = outputString.replace(replacement.search, replacement.replace);
+        }
+
+        return outputString;
+    };
+
+    console.log(station);
```
Comment (avrhamNeeman) [inline ui_src/src/domain/stationsList/stationBoxOverview/index.js:78]: remove console.log

File: ui_src/src/domain/stationsList/stationBoxOverview/index.js [hunk 4]
```diff
@@ -56,38 +60,29 @@ const StationBoxOverview = ({ station, handleCheckedClick, isCheck }) => {
         }
     }, []);
 
+    const replaceTimeUnits = (inputString) => {
```
Comment (avrhamNeeman) [inline ui_src/src/domain/stationsList/stationBoxOverview/index.js:63]: You don't need this function; you can change it directly inside the convertor function "convertSecondsToDate."

File: ui_src/src/domain/stationsList/stationBoxOverview/style.scss [hunk 5]
```diff
@@ -1,20 +1,23 @@
 .station-box-container {
     width: calc(100%);
     background-color: white;
-    border-radius: 10px;
-    box-shadow: 0px 0px 4px 3px rgba(221, 221, 221, 0.3);
+    // border-radius: 10px;
```
Comment (avrhamNeeman) [inline ui_src/src/domain/stationsList/stationBoxOverview/style.scss:4]: remove comments

File: ui_src/src/assets/images/timeSchedule.svg
Comment (aydin101) [reply ui_src/src/assets/images/timeSchedule.svg:1]: If you check the design there is a small difference between the current icon and retentionIcon.svg. retentionIcon.scg is missing clock inside it.

File: ui_src/src/domain/stationsList/stationBoxOverview/index.js (hunk 2 above)
Comment (aydin101) [reply ui_src/src/domain/stationsList/stationBoxOverview/index.js:31]: No, It does not work in that way. When used that way you cannot use it as standalone component. it is imported as path

File: ui_src/src/domain/stationsList/stationBoxOverview/index.js [hunk 6]
```diff
@@ -17,21 +17,23 @@ import { MinusOutlined } from '@ant-design/icons';
 import { Link } from 'react-router-dom';
 import Lottie from 'lottie-react';
 
-import { convertSecondsToDate, isCloud, replicasConvertor } from '../../../services/valueConvertor';
+import { convertSecondsToDate, isCloud, parsingDate, replicasConvertor, parsingDateWithotTime } from '../../../services/valueConvertor';
```
Comment (avrhamNeeman) [inline ui_src/src/domain/stationsList/stationBoxOverview/index.js:20]: Please remove the import of parsingDateWithotTime if you are not using it

**Pull request merged**